package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@flowforge/flowforge",
-    "version": "1.8.0",
+    "version": "1.8.1",
```

Comment:
We already have a 1.8.1 for the nr-launcher and helm, do we want to go to 1.8.2?

Reply:
ok - will make it so